src/Tilesets/EffectsTileset.tscn

```diff
@@ -1,9 +1,3 @@
-[gd_scene load_steps=2 format=2]
-
-[ext_resource path="res://Assets/effects/energy_shield.png" type="Texture" id=1]
+[gd_scene format=2]
```

Comment:
Do we even need this tileset any more?

Reply:
Yeah. Wouldn't make sense to use the humans tileset in the effects tilemap.